schemas/json/layout/layout.schema.v1.json

```diff
@@ -873,12 +873,14 @@
       "type": "object",
       "properties": {
         "tableHeaders": {
-          "type": "array",
-          "items": {
-            "type": "string"
-          },
+          "type": "object",
           "title": "Table Headers",
-          "description": "An array of strings that is going to be headers of the table. Can be added to the resource files to change between languages"
+          "examples": [
+            {
+              "AFieldInRowDataModel": "header1",
+              "OtherFieldInRowDataModell": "header2"
+            }],
+          "description": "An object where the fields in the datamodel is mapped to headers. Must correspond to datamodel representing a row. Can be added to the resource files to change between languages"
```

Comment:
"Datamodel" means something very different, so we should use a different word. This about keys in the DataList, right? If I saw the example for the first time, it would be difficult for me to guess if the key or value here is the key in DataList, and which of them is the label the user will read. How about this example instead?

```json
{
  "productId": "product.id",
  "description": "Beskrivelse av produkt"
}
```